src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { HeaderGenerator, HeaderGeneratorOptions, headerGeneratorOptionsShape } from './header-generator';
```

Comment:
might be safer to do full reexport, e.g. if you add new interface/type to that file, it would be automatically exported

```suggestion
export * from './header-generator';
```